### ISSUE TITLE
Extension methods for RTCIceTransports from PeerConnection

### ISSUE THREAD

**Issue by steveanton** [w3c/webrtc-ice#14]
How do the extension methods defined in this specification apply to the RTCIceTransports returned from PeerConnection (via RTCDtlsTransport)?

Possible ideas:
* Throw an exception if the object is from a PeerConnection.
* Do not include the methods in the prototype of the RTCIceTransport returned from RTCDtlsTransport (is this valid?).
* Specify how the methods interact with the offer/answer model of PeerConnection.

**Comment by aboba** [comment]
I don't think it makes sense for `RTCIceTransport` extension methods like `gather()` or `start()` to be called on the `RTCIceTransport` vended by an `RTCPeerConnection`.  So I would not attempt to specify how the extension methods interaction with the O/A model - an exception seems like the right behavior.

**Comment by aboba** [comment]
@steveanton In the recent [checkins to support ICE forking](https://webrtc-review.googlesource.com/c/src/+/162241), there [seems to be a desire to support ICE forking with `RTCIceTransports` from `RTCPeerConnection`.](https://webrtc-review.googlesource.com/c/src/+/162241/11/pc/peer_connection_ice_unittest.cc) 

Is that right?

**Comment by steveanton** [comment]
Yeah, does appear to be the case.
